frontend/stylesheets/_dashboard.less

```diff
@@ -6,12 +6,20 @@
 
 .gallery-dataset-col {
   padding-bottom: 24px;
+  width: 100%;
 
   @media @smartphones {
     padding-bottom: 32px;
   }
 }
 
+.not-highlighted-link:link,
+.not-highlighted-link:visited,
+.not-highlighted-link:hover,
+.not-highlighted-link:active {
+  text-decoration: none !important;
```

Comment:
As we're using less, I think you should be able to write this a little more concise, like:
```less
.not-highlighted-link {
  &:link, &:visited, &:hover, &:active {
    text-decoration: none !important;
  }
}
```